client/docs/register.md

```diff
@@ -93,6 +100,13 @@
         show_root_heading: false
         show_root_toc_entry: false
 
+### ClickHouse
+::: featureform.register.Registrar.register_clickhouse
+    handler: python
+    options:
+        show_root_heading: false
+        show_root_toc_entry: false
+
```

Comment:
Perhaps I'm missing something here, but I'm not seeing why there needs to be 2 entires for ClickHouse here.